binding/HarfBuzzSharp.Shared/Buffer.cs

```diff
@@ -44,15 +43,49 @@ public uint Length
 			get { return HarfBuzzApi.hb_buffer_get_length(Handle); }
 		}
 
-		public void AddUtf8(string utf8text)
+		public void AddUtf8(string text)
 		{
-			var bytes = Encoding.UTF8.GetBytes(utf8text);
+			var bytes = Encoding.UTF8.GetBytes(text);
 			AddUtf8(bytes);
 		}
 
-		public void AddUtf8(byte[] bytes)
+		public void AddUtf8(byte[] text, uint itemOffset = 0, int itemLength = -1)
 		{
-			HarfBuzzApi.hb_buffer_add_utf8(Handle, bytes, bytes.Length, 0, bytes.Length);
+			if (itemLength == -1) {
+				itemLength = text.Length - (int)itemOffset;
+			}
```

Comment:
`hb_buffer_add_*` do this internally, so you can just pass the -1 to them.

Reply:
Thanks for letting me know that. Will change it accordingly 👍